layers/+lang/go/packages.el

```diff
@@ -129,6 +129,7 @@
         "ia" 'go-import-add
         "ig" 'go-goto-imports
         "ir" 'go-remove-unused-imports
+        "tb" 'spacemacs/go-run-package-benchmarks
```

Comment:
The `CONVENTIONS.org` file requires that `tb` is reserved for running all the tests in the buffer. For benchmarks a new keybinding must be used. Please look at how other layers define a similar functionality.